### PR TITLE
Fix typing isues with latest typedefs

### PR DESCRIPTION
**Pull request by vishalshrm539**
* Fixed typing issues that came up with latest typedefs.
* Updated typedefs package version to latest.
* Added a script to pack the overrides.

### CODE REVIEW

**Review by paulgagnon-pega — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "prebuild-overrides": "shx rm -rf ./packages/react-sdk-overrides/lib && shx cp -r ./packages/react-sdk-components/src/components ./packages/react-sdk-overrides/lib",
     "build-overrides": "node scripts/build-overrides",
     "postbuild-overrides": "echo build-overrides completed",
+    "pack-overrides": "npm  pack ./packages/react-sdk-overrides",
```
Comment (tumms2021389) [inline package.json:15]: What is the need of this script? Let's not have too many scripts

Comment (tumms2021389) [reply package.json:15]: Lets remove it for now, we can check this later

**Review by tumms2021389 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (vishalshrm539) [reply package.json]: We already have a similar script for packing the SDK(i.e. pack-sdk) so can have similiar for overrides I believe.

Comment (tumms2021389) [reply package.json]: pack-sdk doesn't give any meaning to me, lets remove the scripts for packing from package.json as we don't run them more often and our document already has the steps for packaging the components and overrides

Comment (vishalshrm539) [reply package.json]: Ok, got rid of both.

**Review by tumms2021389 — APPROVED** [review]

**Pull request merged**